tools/props/VersionsNet8OrLater.props

```diff
@@ -5,5 +5,8 @@
     <SystemConfigurationConfigurationManagerVersion>8.0.0</SystemConfigurationConfigurationManagerVersion>
     <SystemRuntimeCachingVersion>8.0.0</SystemRuntimeCachingVersion>
     <MicrosoftExtensionsCachingMemoryVersion>8.0.0</MicrosoftExtensionsCachingMemoryVersion>
+    <MicrosoftDotNetRemoteExecutorVersion>9.0.0-beta.24157.1</MicrosoftDotNetRemoteExecutorVersion>
+    <MicrosoftDotNetXUnitEtensionsVersion>9.0.0-beta.24157.1</MicrosoftDotNetXUnitEtensionsVersion>
```

Comment:
I think line 9 is redundant.

Reply:
Maybe not redundant, but I suppose outside the scope, since we don't need to upgrade xunit extensions today.

Reply:
@JRahnama How's it look now?

Reply:
@benrr101 LGTM. I will do some testing today and will approve it after. Thank you.